keps/sig-node/4576-conditions-for-pod-terminated/kep.yaml

```
@@ -0,0 +1,42 @@
+title: Pod terminated condition
+kep-number: 4576
+authors:
+  - "@DrAuYueng"
+owning-sig: sig-node
+
+participating-sigs:
+  - sig-node
```

```suggestion
  - sig-storage
```
If we consider the volume as an important use case.